pages/common/w3m.md

```diff
@@ -0,0 +1,11 @@
+# w3m
+
+> A text based Web browser and pager.
```

Comment:
I believe "text-based" is the correct form. Also, the "...and pager" is probably more puzzling than helpful to beginners. Is it really needed?